src/farcaster/botCast.ts

```
@@ -0,0 +1,43 @@
+  makeCastAdd,
+  NobleEd25519Signer,
+} from "@farcaster/hub-nodejs";
+import { submitMessage } from "./submitMessage.js";
+
+const SIGNER_PRIVATE_KEY = process.env.SIGNER_PRIVATE_KEY as Address;
+const APP_FID = Number(process.env.APP_FID);
+
+if (!SIGNER_PRIVATE_KEY) {
+  throw new Error("SIGNER_PRIVATE_KEY is not set");
+}
+if (!APP_FID || isNaN(APP_FID)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)